SU2_CFD/include/drivers/CDiscAdjSinglezoneDriver.hpp

```diff
@@ -122,8 +208,114 @@ class CDiscAdjSinglezoneDriver : public CSinglezoneDriver {
    */
   void Run(void) override;
 
+  /*!
+   * \brief Update the discrete adjoint solution.
+   */
+  void UpdateAdjoints(void);
+
+  /*!
+   * \brief Update the primal time iteration.
+   */
+  void UpdateTimeIter(void);
+
+  /*!
+   * \brief Update the primal far-field variables.
+   */
+  void UpdateFarfield(void);
+
+  /*!
+   * \brief Update the primal geometry (does not include mesh deformation).
+   */
+  void UpdateGeometry(void);
```

Comment:
Remove the (voids) please.
```suggestion
  void UpdateGeometry();
```